packages/suite/src/utils/suite/formatTimeLeft.ts

```diff
@@ -1,13 +1,26 @@
 import formatDuration from 'date-fns/formatDuration';
 import intervalToDuration from 'date-fns/intervalToDuration';
+import isDate from 'date-fns/isDate';
 
 export const formatTimeLeft = (
     deadline: Date,
     locale?: Locale,
     format: Array<keyof Duration> = ['hours', 'minutes'],
 ) => {
+    if (!deadline || !isDate(deadline)) {
+        return '';
+    }
+
+    const currentTime = Date.now();
+
+    const isPastTheDeadline = deadline.getTime() < currentTime;
+
+    if (isPastTheDeadline) {
+        return formatDuration({ minutes: 0 }, { locale, format: ['minutes'], zero: true });
+    }
```

Comment:
i would rather wrap intervalToDuration into try/catch and return 0 or empty string for all possible runtime errors like:
```
try {
        const duration = intervalToDuration({
            start: Date.now(),
            end: deadline,
        });
        const formattedTimeLeft = formatDuration(duration, { locale, format });

        return formattedTimeLeft;
    } catch (error) {
        return ''; // or 0 time left
    }
```

Reply:
I wanted to distinguish between a wrong input and no time left. I can of course have a try/catch